spec/fixtures/smartctl/sda_h

```diff
@@ -0,0 +1,5 @@
+smartctl 6.2 2013-07-26 r3841 [x86_64-linux-3.19.0-68-generic] (local build)
+Copyright (C) 2002-13, Bruce Allen, Christian Franke, www.smartmontools.org
+
+=== START OF READ SMART DATA SECTION ===
+SMART overall-health self-assessment test result: PASSED
```

Comment:
Probably better to name these `spec/fixtures/smartctl/passed` and `spec/fixtures/smartctl/failed`